src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -1840,6 +1840,13 @@ public void testArgumentOrderOfErrorMessages() {
                 isInOrder);
     }
 
+    @Test
+    public void testEmptyArray() throws Exception {
+        final DefaultConfiguration checkConfig = createCheckConfig(IndentationCheck.class);
+        final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
+        verify(checkConfig, getPath("InputIndentationEmptyArray.java"), expected);
```

Comment:
Indentation is very special as it does alot of extra checks and balances because of it's complexity.
Please change `verify` to `verifyWarns` to turn these checks on.